docs/js/alert.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Alert
+layout: page.jade
+sidebar: true
+collection: js
+priority: 3
+path: alert
+---
+
+# Alert
+<p class="lead">The Alert component is an empty container where you can add a message that will be presented to users. It has an OK button by default.</p>
+
+An alert box is often used when you want to make sure information comes through to the user.
+
+### How to use
+
+You can initiate the component as a jQuery plugin:
+
+```js
+// using any selector
+$('any-selector').on('click', () => $.fn.alert(options));
+```
+
+or a vanilla constructor:
+
+```js
+import Alert from 'garden/src/js/components/alert';
+
+new Alert(options);
+```
+
+## Options
+
+Alert provides some customizable options such as: `textMessage`, `textButton`, `size`, and `triggerClose`. The default value for the `textMessage` is 'This is an example message', for the `textButton` is 'Ok', the `size` value is 'medium', and `triggerClose` is 'data-alert-button'.
+
+| Option            | Default | Description |
+|-------------------|-------------|
+| textMessage  | `This is an example message` | A text to display in the alert |
+| textButton | `Ok` | A text to alert button |
```

Comment:
Aqui fica meio confuso se `alert` é verbo ou substantivo/adjetivo. Coloca um `the` depois do `to`